BookingsApi/BookingsApi.DAL/Migrations/20211210101915_AddCaseAndRoleTypeForUpperTaxTribunal.cs

```diff
@@ -0,0 +1,80 @@
+﻿using BookingsApi.DAL.Helper;
+using BookingsApi.Domain.Enumerations;
+using BookingsApi.Domain.RefData;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddCaseAndRoleTypeForUpperTaxTribunal : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
+                nameof(CaseType),
+                new[] { "Id", "Name" },
+                new object[,]
+                {
+                    {39, "Upper Tribunal Tax"}
+                });
+
+            migrationBuilder.InsertData(
+                nameof(HearingType),
+                new[] { "Id", "Name", "CaseTypeId" },
+                new object[,]
+                {
+                    {178, "Permission to Appeal", 39}
+                });
+
+            migrationBuilder.InsertData(
+                nameof(CaseRole),
+                new[] { "Id", "Name", "Group", "CaseTypeId" },
+                new object[,]
+                {
+                    { 251, "Appellant", (int) CaseRoleGroup.Appellant, 39 },
+                    { 252, "Respondent", (int) CaseRoleGroup.Respondent, 39 },
+                    { 253, "Panel Member", (int) CaseRoleGroup.PanelMember, 39 },
+                    { 254, "Observer", (int) CaseRoleGroup.Observer, 39 },
+                    { 260, "Judge", (int) CaseRoleGroup.Judge, 39 }
```

Comment:
Why is there a gap between 254 and 260?

Reply:
I took over this dev - Judge was missing in this case role, it was detected when running integration test and other the id's are already taken for other case type.